File: PHP/Car.php
<?php
class Car {
  public $id;
  public $license;
  public $driver;
  public $passengers;

  public function __construct($license, $driver) {
    $this->license = $license;
    $this->driver = $driver;
  }

  public function printCar() {
    echo " => Lincese: " .$this->license. "\n";
    echo " => Driver: " .$this->driver->name. "\n";
  }
}
?>

File: Java/Car.java
package Java;

public class Car {
  Integer id;
  String license;
  Account driver;
  private Integer passenger;

  public Car(String license, Account driver) {
    this.license = license;
    this.driver = driver;
  }

  void printDataCar() {
    if (passenger != null) {
      System.out.println("License: " + license + " Driver Name: " + driver.name);
    }
  }

  public Integer getPassenger() {
    return passenger;
  }

  public void setPassenger(Integer passenger) {
    if (passenger == 4) {
      this.passenger = passenger;
    } else {
      System.out.println("Need to assign 4 passengers");
    }
  }
}
